### PR TITLE
Added index_select to paddle frontend search mod

### LINKED ISSUE #20840: index_select

### PR DESCRIPTION
**Pull request by nikita-jare**
Close #20840

Added index_select to paddle frontend search module

### CODE REVIEW

**Review by AryanSharma21 — COMMENTED** [review]
Hello, @nikita-jare , as per the test result, there is some argument error in the function that you have implemented. 
Please check by your side .

E   TypeError: index_select() got multiple values for argument 'index'

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hi, thanks for looking into this. Getting this error on testing - 
```
TypeError: index_select() got multiple values for argument 'index'
```
I looked through your code and couldn't find anything wrong - except for hard-coding array shapes (added comment). Could you try syncing this branch with latest updates?

Feel free to re-request review after fixing, thanks! 😄

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py [hunk 1]
```diff
@@ -206,3 +206,36 @@ def test_paddle_searchsorted(
...
     )
+
+
+# index_select
+@handle_frontend_test(
+    fn_tree="paddle.index_select",
+    params_indices_others=helpers.array_indices_axis(
+        array_dtypes=helpers.get_dtypes("valid"),
+        indices_dtypes=["int32", "int64"],
+        max_num_dims=1,
```
Comment (rishabgit) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py:217]: this would hard-code shape of inputs too which isn't ideal

**Comment by rishabgit** [comment]
This PR has been closed because it has been stale for more than 7 days with no activity.

**Pull request closed without merging**